src/applications/find-forms/tests/components/FormTitle.unit.spec.jsx

```diff
@@ -0,0 +1,53 @@
+// Dependencies.
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+// Relative imports.
+import FormTitle from '../../components/FormTitle';
+
+describe('Find VA Forms <FormTitle />', () => {
+  const props = {
+    id: 'VA10192',
+    formToolUrl:
+      'https://www.va.gov/health-care/about-information-for-pre-complaint-processing/',
+    title: 'Information for Pre-Complaint Processing',
+  };
+
+  it('should expect props', () => {
+    const tree = mount(
+      <FormTitle
+        id={props.id}
+        formUrl={props.formToolUrl}
+        title={props.title}
+      />,
+    );
+
+    // Props testing
+    expect(tree.props().id).to.equal(props.id);
+    expect(tree.props().formUrl).to.equal(props.formToolUrl);
+    expect(tree.props().title).to.equal(props.title);
+
+    tree.unmount();
+  });
+
+  it('should render', () => {
+    const tree = mount(
+      <FormTitle
+        id={props.id}
+        formUrl={props.formToolUrl}
+        title={props.title}
+      />,
+    );
+    const treeText = tree.text();
+
+    // expecting result node tree text to include the following
+    [(props.title, props.id)].forEach(text => {
+      expect(treeText).to.include(text);
+    });
```

Comment:
Is this different from this?

```
expect(treeText).to.include(props.title);
expect(treeText).to.include(props.id);
``` 

If you prefer the `.forEach`, should the parentheses be nested in the array brackets?

Reply:
Yeah, the forEach was already there. I personally don't care too much what we use. A loop or just simple list of statements. 

As far as the parentheses is concerned, I think Prettier added those via On Save formatting? I'm not sure. Will Investigate.

Reply:
Yeah I was wondering that too. I wasn't sure how the parentheses may change how the values are processed. It looks like only the last truth-y element is returned?

![image](https://user-images.githubusercontent.com/1915775/101504490-48162380-3941-11eb-9a7e-7c05fccd5a15.png)

Reply:
Yeah, it does appear that way. That's interesting.